2200056/14654 신용카드 만들기2.py

```diff
@@ -0,0 +1,21 @@
+T = int(input())
+num_list = ["3", "4", "5", "6", "9"] # 카드번호 앞자리 조건 리스트 만들기
+for t in range(1, T+1):
+    credit_card = input()
+    
+    if len(credit_card) == 16: # 카드번호가 16자리인 경우 카드번호 조건
+        if credit_card[0] in num_list: # 그 중에서 앞자리 조건 리스트를 만족 시킬 경우
+            print(f"#{t} 1")
+        else:
+            print(f"#{t} 0")
+
+    elif len(credit_card) > 16: # 16자리 이상인 경우 (-가 있는 경우)
+        for num in credit_card:
+            if num == "-": # 카드번호에 -가 있는 경우
+                new_credit_card = credit_card.replace(num, "") # -를 공백으로 바꾸기
```

Comment:
조건을 잘 나눠서 풀어주셨네요 👍 
```suggestion
        new_credit_card = credit_card.replace("-", "") # -를 공백으로 바꾸기
```
replace 함수가 string의 모든 "-" 를 공백으로 바꿔주므로, 한 번만 실행하셔도 됩니다 😄